### DISCUSSION TITLE
Labels.

### DISCUSSION THREAD

**Discussion by krstp** [tutao/tutanota#6651] [Ideas]
Since I decided one more time to support Tutanota, even more, and pay for another tier, I believe this is the right time to post my grievance.

Tutanota has been on the market for quite a long time. As such, users making request for labels/tags has been notorious. In response, I just find Tutanota team somewhat ignorant (if not arogant) to the issue, simply closing some of the ideas/issues requests as `closed for discussion`, simply blocking comments, however, issue remaining open with idle status, with users not even being able to LIKE or DISLIKE the post: https://github.com/tutao/tutanota/issues/318. IMHO, this is somewhat outrageous. Side note: I know there were more _labeling request_ posts, but somewhat they... disappeared! - where is transparency I ask :) 

Tuta, despite lacking some of the functionality, has delivered outstanding email service, HOWEVER, this is unheard of to just ignore  and block user base requests for simple feature such as email labels.

So in hope to spark some action from the Tutanota team, at the same time committing to another tier of service, I would like to make not a unique or anyhow outstanding, but seriously demanding, a kind request to enable email **labels** within Tutanota service. I am sure some will see it as poking stick into the ant-mound, so be it! We NEED and WANT labels :) 

PS. To resonate on the past read conversations, YES, folders ARE NOT equivalent to labels, and YES, emails can have MULTIPLE LABELS; and having ability to use folders DOES NOT solve the labeling feature - I write this because I have read this response from Tuta devs quite a few times over the period of last few years and in a weird stubborn way I kept reading the same :) 

Seriously, by this time, I really hopped we could have labeling feature :)

Kind regards to the team, thanks for all the work you do, but truth to be told, you guys seriously suck on listening to users and their calls for labels.

**Comment by charlag** [comment]
Hi, the labels are scheduled. They are also on the [roadmap](https://tuta.com/roadmap)

**Reply by krstp** [reply]
Yeah, roadmap references the same issue I posted above, which is 6 years old :) Either way, glad to know it is on the roadmap.

**Reply by charlag** [reply]
age of issues is not an indicator of anything :)

**Comment by dermitdemdino** [comment]
Label should be available soon. Is there a beta program you can join?

**Comment by igorpupkinable** [comment]
I guess this discussion can be closed.
